cleansio/audio/audio_file.py

```diff
@@ -20,29 +20,28 @@ def __init__(self, file_path):
         self.channels = audio_segment.channels
         self.frame_rate = audio_segment.frame_rate
         self.chunk_length = 5000 # In milliseconds
-        self.normal_chunks, self.overlapping_chunks = \
-            self.__init_create_chunks(audio_segment)
+        self.normal_chunks = self.__init_create_chunks(audio_segment)
 
     def __init_create_chunks(self, audio_segment):
         """ Breaks up the file into small chunks """
         temp_dir = create_temp_dir()
         normal_chunks = []
-        overlapping_chunks = []
         # Create normal and overlapping chunks
         self.__create_chunks(audio_segment, temp_dir, normal_chunks, False)
-        self.__create_chunks(audio_segment, temp_dir, overlapping_chunks, True)
+        self.__create_chunks(audio_segment, temp_dir, [], True)
         # Add the list of chunk filepaths to an ENV variable for post cleanup
         create_env_var(
-            'CLEANSIO_CHUNKS_LIST', str(normal_chunks + overlapping_chunks))
```

Comment:
You can't remove this. This env variable keeps track of which files we need to delete on cleanup. If you remove it the overlapping-accuracy chunk files will stay on the system forever. Check `~/.cleansio-temp`